contributor_docs/hi/steward_guidelines.md

```diff
@@ -0,0 +1,351 @@
+# स्टीवर्ड दिशानिर्देश
+
+चाहे आप हाल ही में हमारे साथ स्टीवर्ड के रूप में शामिल हुए हों, p5.js के अनुभवी रखरखावकर्ता हों, या कहीं बीच में हों, यह गाइड जानकारी के साथ-साथ उन सुझावों और ट्रिक्स को शामिल करता है जो आपको p5.js में प्रभावी योगदान देने में मदद करेगा। यहां लिखा गया अधिकांश दिशानिर्देश हैं, अगर कुछ और नहीं बताया गया है, तो इसका अर्थ है कि आप यहां दिखाए गए अभ्यासों को अपने काम के लिए अनुकूल बना सकते हैं।
+
+## विषय सूची
+
+- [समस्याएँ](./steward_guidelines.md#समस्याएँ)
+
+  - [बग रिपोर्ट](./steward_guidelines.md#बग-रिपोर्ट)
+  - [सुविधा अनुरोध](./steward_guidelines.md#सुविधा-अनुरोध)
+  - [सुविधा विस्तार](./steward_guidelines.md#सुविधा-विस्तार)
+  - [चर्चा](./steward_guidelines.md#चर्चा)
+
+- [पुल-रिक्वेस्ट](./steward_guidelines.md#पुल-रिक्वेस्ट)
+
+  - [सरल सुधार](./steward_guidelines.md#सरल-सुधार)
+  - [बग फ़िक्स](./steward_guidelines.md#बग-फ़िक्स)
+  - [नई सुविधा/सुविधा वृद्धि](./steward_guidelines.md#नई-सुविधासुविधा-वृद्धि)
+  - [डिपेंडेबॉट](./steward_guidelines.md#डिपेंडेबॉट)
+
+- [निर्माण प्रक्रिया](./steward_guidelines.md#निर्माण-प्रक्रिया)
+
+  - [मुख्य निर्माण कार्य](./steward_guidelines.md#मुख्य-निर्माण-कार्य)
+  - [विविध कार्य](./steward_guidelines.md#विविध-कार्य)
+
+- [युक्तियाँ और ट्रिक्स](./steward_guidelines.md#युक्तियाँ-और-ट्रिक्स)
+
+  - [उत्तर टेम्पलेट](./steward_guidelines.md#उत्तर-टेम्पलेट)
+  - [गिटहब सीएलआई](./steward_guidelines.md#गिटहब-सीएलआई)
+  - [सूचनाओं का प्रबंधन](./steward_guidelines.md#सूचनाओं-का-प्रबंधन)
+
+---
+
+## समस्याएँ
+
+हम अधिकांश स्रोत कोड योगदानों को एक समस्या के साथ शुरू करने की प्रोत्साहना करते हैं, और इस तरह, समस्याओं में अधिकांश चर्चाएँ होंगी। समस्या को समीक्षा करने के लिए लेने के चरण वास्तव में यह निर्भर करेगा कि यह कैसी समस्या है। रेपो विभिन्न प्रकार की समस्याओं को बेहतर ढंग से व्यवस्थित करने और समस्या लेखकों को अपनी समस्याओं के बारे में सभी प्रासंगिक जानकारी प्रदान करने के लिए [गिटहब समस्या टेम्पलेट](https://github.com/processing/p5.js/blob/main/.github/ISSUE_TEMPLATE) का उपयोग करता है। समस्या की समीक्षा की पहली कदम अक्सर भरे गए टेम्पलेट को देखना होगा और यह तय करना होगा कि क्या आपको अतिरिक्त जानकारी की आवश्यकता है (जैसे कि कुछ क्षेत्र भरे नहीं गए हों या गलत टेम्पलेट का प्रयोग किया गया हो)।
```

Comment:
और इस तरह, समस्याओं में अधिकांश चर्चाएँ होंगी - this isn't the meaning conveyed in the English docs, request you to change the same

Reply:
Thanks for the review!